Util/Tokenization/src/main/java/org/tribuo/util/tokens/impl/wordpiece/WordpieceTokenizer.java

```diff
@@ -0,0 +1,205 @@
+package org.tribuo.util.tokens.impl.wordpiece;
+
+import java.text.Normalizer;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.Set;
+
+import org.tribuo.util.tokens.Token;
+import org.tribuo.util.tokens.Token.TokenType;
+import org.tribuo.util.tokens.Tokenizer;
+
+import com.oracle.labs.mlrg.olcut.config.Config;
+import com.oracle.labs.mlrg.olcut.provenance.ConfiguredObjectProvenance;
+import com.oracle.labs.mlrg.olcut.provenance.impl.ConfiguredObjectProvenanceImpl;
+
+public class WordpieceTokenizer implements Tokenizer {
+
+    @Config
```

Comment:
Make this config mandatory, and add descriptions for all of them.

Reply:
done.